src/tlo/methods/healthsystem.py

```diff
@@ -203,6 +203,10 @@ class HealthSystem(Module):
             Types.STRING, "Name of priority policy adopted"),
         'year_mode_switch': Parameter(
             Types.INT, "Year in which mode switch is enforced"),
+         'scale_to_effective_capabilities': Parameter(
```

Comment:
```suggestion
        'scale_to_effective_capabilities': Parameter(
```
To match other indentation